v20.2/performance.md

```diff
@@ -15,44 +15,52 @@ This document is about CockroachDB’s performance on benchmarks. For guidance o
 
-<img src="{{ 'images/v20.2/tpcc100k.png' | relative_url }}" alt="TPC-C 100,000" style="max-width:100%" />
+For a refresher on what exactly TPC-C is and how it is measured, see [Benchmarks used](#benchmarks-used) below.
 
-Metric                             | CockroachDB Results       
------------------------------------|-------------------
-Max Throughput                     | 1,245,462 tpmC    
-Max Warehouses with Max Efficiency | 100,000 Warehouses
-Max Number of Rows                 | 49.8B             
-Max Unreplicated Data              | 8TB               
-Number of Nodes                    | 81
-Node Machine Type                  | c5d.9xlarge            
+(<strong><font style="size:xx-large" color="red">FIXME</font></strong>: Replace this chart with the one provided by Design)
 
+<img src="{{ 'images/v20.2/tpcc140k.png' | relative_url }}" alt="TPC-C 140,000" style="max-width:100%" />
+
+| Metric                             | CockroachDB Results |
+|------------------------------------+---------------------|
+| Max Throughput                     | 1,684,437 tpmC      |
+| Max Warehouses with Max Efficiency | 140,000 Warehouses  |
+| Max Number of Rows                 | 49.8B               |
+| Max Unreplicated Data              | 8TB                 |
+| Number of Nodes                    | 81                  |
+| Node Machine Type                  | c5d.9xlarge         |
 
 CockroachDB achieves this performance in [`SERIALIZABLE` isolation](demo-serializable.html), the strongest isolation level in the SQL standard.
+
 ### Linear scaling
 
 CockroachDB has **no theoretical scaling limit** and, in practice, can achieve near-linear performance at 256 nodes. Because the TPC-C results above reflect leaps in scale, to test linear scaling, Cockroach Labs ran a simple benchmark named KV 95 (95% point reads, 5% point writes, all uniformly distributed) on AWS `c5d.4xlarge` machines:
 
+<strong><font style="size:xx-large" color="red">FIXME</font></strong>: Update this chart with latest data - see <https://github.com/cockroachdb/docs/issues/8212>
+
 <img src="{{ 'images/v20.2/linearscale.png' | relative_url }}" alt="CRDB Linear Scale" style="max-width:100%" />
 
 This chart shows that adding nodes increases throughput linearly while holding p50 and p99 latency constant. The concurrency for each scale was chosen to optimize throughput while maintaining an acceptable latency and can be observed in the table below.
 
-| Number of Nodes | Workers | Concurrency
-|-----------------|---------|------------|
-|       16        |    2    |   512
-|       32        |    4    |   512
-|       64        |    4    |   1024
-|       128       |    8    |   1024
-|       256       |    8    |   2048
+| Number of Nodes | Workers | Concurrency |
```

Comment:
I assume this table would also need to be updated if we have new numbers in the chart above.